src/sentry/models/group.py

```diff
@@ -671,6 +671,7 @@ class Group(Model):
     priority_locked_at = models.DateTimeField(null=True)
     seer_fixability_score = models.FloatField(null=True)
     seer_autofix_last_triggered = models.DateTimeField(null=True)
+    seer_automation_queued = models.BooleanField(default=False)
```

Comment:
Is this something that will only happen once? Or will this be set back to false after the automation runs?

I'm wondering if we can just use a redis key to stop the automation being run too many times. It depends whether this data is useful to us long term though

Reply:
Ok, in that case I think it's fine to keep this on the group as a column. I'm starting to wonder if we should break out the seer values into a new table at some point, just because this table is getting quite wide. More of a discussion if we're going to keep adding columns though

Reply:
Nice, the fewer columns the better. Thanks for looking into this